common/disabled/dataCodec_test.go

```diff
@@ -0,0 +1,92 @@
+package disabled
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDataCodec_MethodsShouldNotPanic(t *testing.T) {
+	t.Parallel()
+
+	dc := NewDisabledDataCodec()
+	require.False(t, check.IfNil(dc))
+
+	require.NotPanics(t, func() {
+		_, _ = dc.SerializeEventData(sovereign.EventData{})
```

Comment:
I actually meant to test the result in here, not below with a test for each func.